cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,19 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+		Example: `# Create a Cluster Workflow Template with Default Options:
```

Comment:
```suggestion
		Example: `# Create a Cluster Workflow Template:
```

- "default options" is implied since no options were given
- "Default Options" should not be capitalized as it is not a name / proper noun